src/components/Alerts/Alerts.README.md

```diff
@@ -0,0 +1,80 @@
+# Alerts API
+
+Alert can be accessible/render by Alert API, you can import alertAPI from `Alerts.tsx`.
+The API have only one function `open`.
+
+## Usage
+
+```javascript
+import alertAPI as AlertAPI from "./index"
+
+alertAPI.open({
+    type="warning",
+    size="small",
+    content="Hello There, this should the multiline content",
+    autoDismiss=true,
+    dismissDuration = 3000,
+    heading = "This is heading of the multiline alert",
+    multiline="true"
+})
+```
+
+### Parameter of`.open()`
+
+You can pass props value of Alert Component as object, which are mentioned in interface `AlertProps`
+
+# Alert
```

Comment:
Since this is not exposed, it should not be there in the readme either. The props descriptions should instead be changed to reflect the object properties that will be passed as the argument.